### PR TITLE
Add bindings for setting and getting emulation mouse from touch and touch from mouse in input

### PR DESCRIPTION
**Pull request by CaTaTo23**
This will allow to set emulation of this input events in runtime, for example enabling it in game menu on touchscreen only device to allow use Control based buttons

### CODE REVIEW

File: core/input/input.cpp [hunk 1]
```diff
@@ -143,6 +143,10 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_use_accumulated_input", "enable"), &Input::set_use_accumulated_input);
 	ClassDB::bind_method(D_METHOD("is_using_accumulated_input"), &Input::is_using_accumulated_input);
 	ClassDB::bind_method(D_METHOD("flush_buffered_events"), &Input::flush_buffered_events);
+	ClassDB::bind_method(D_METHOD("set_emulate_mouse_from_touch", "p_emulate"), &Input::set_emulate_mouse_from_touch);
+	ClassDB::bind_method(D_METHOD("is_emulating_mouse_from_touch"), &Input::is_emulating_mouse_from_touch);
+	ClassDB::bind_method(D_METHOD("set_emulate_touch_from_mouse", "p_emulate"), &Input::set_emulate_touch_from_mouse);
+	ClassDB::bind_method(D_METHOD("is_emulating_touch_from_mouse"), &Input::is_emulating_touch_from_mouse);
```
Comment (akien-mga) [inline core/input/input.cpp:149]:
```suggestion
	ClassDB::bind_method(D_METHOD("set_emulate_mouse_from_touch", "enable"), &Input::set_emulate_mouse_from_touch);
	ClassDB::bind_method(D_METHOD("is_emulating_mouse_from_touch"), &Input::is_emulating_mouse_from_touch);
	ClassDB::bind_method(D_METHOD("set_emulate_touch_from_mouse", "enable"), &Input::set_emulate_touch_from_mouse);
	ClassDB::bind_method(D_METHOD("is_emulating_touch_from_mouse"), &Input::is_emulating_touch_from_mouse);
```

**Review by akien-mga — COMMENTED** [review]
You should bind those as properties (`ADD_PROPERTY` below, see how it's done for `use_accumulated_input`), and then sync the class reference by running your compiled Godot with `--doctool` to generate the documentation template (and fill the description).

File: core/input/input.cpp (hunk 1 above)
Comment (KoTeYkA23) [reply core/input/input.cpp:149]: Are this property name only visual or this is reference to function properties?

**Comment by KoTeYkA23** [comment]
I think i messed up something seriously, it will be easier to create new pr, sorry about that

**Comment by AThousandShips** [comment]
Make sure to open your new PR on a dedicated branch and make your changes there and not `master`

**Pull request closed without merging**